doc/javascript_api.md

```diff
@@ -97,7 +97,7 @@ tokenDefinition = {
         attributeId: attribute,
         category: {
             name: "Cat",
-	    syntax: "IA5String",
```

Comment:
@colourful-land this is the example, since CAT could also conceivably be an Integer, I thought it would be best to airbrush the type.